### PR TITLE
HOCS-3449 Swapping to ex-gratia specific business contributions

### PR DESCRIPTION
**Pull request by gardnersam**

### CODE REVIEW

File: src/main/resources/processes/EXGRATIA_ESCALATE.bpmn [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <bpmn:outgoing>SequenceFlow_0aedxp3</bpmn:outgoing>
       <bpmn:outgoing>SequenceFlow_0y3mwb3</bpmn:outgoing>
     </bpmn:exclusiveGateway>
-    <bpmn:serviceTask id="Screen_Input" name="Input" camunda:expression="COMP_SERVICE_ESCALATE_INPUT" camunda:resultVariable="screen">
+    <bpmn:serviceTask id="Screen_Input" name="Input" camunda:expression="EX_GRATIA_ESCALATE_INPUT" camunda:resultVariable="screen">
```
Comment (philbarton) [inline src/main/resources/processes/EXGRATIA_ESCALATE.bpmn:12]: Is this covered by tests?

**Review by philbarton — CHANGES_REQUESTED** [review]
LGTM - just note about tests

File: src/main/resources/processes/EXGRATIA_ESCALATE.bpmn (hunk 1 above)
Comment (gardnersam) [reply src/main/resources/processes/EXGRATIA_ESCALATE.bpmn:12]: There is 100% coverage for these ex-gratia workflows, as the id didn't change the test coverage was maintained. I've added more explicit coverage for the this screen in the most recent commit

**Review by philbarton — APPROVED** [review]
LGTM

**Pull request merged**